app/javascript/mastodon/locales/defaultMessages.json

```diff
@@ -4481,4 +4481,4 @@
     ],
     "path": "app/javascript/packs/public.json"
   }
-]
\ No newline at end of file
+]
```

Comment:
@ClearlyClaire my editor is adding a new line automatically to this file.. I think it makes sense to have it, but I can remove it if we don't want to change it.

Reply:
Yeah, the file is autogenerated, and should not have a newline at the end.

Reply:
Ok I think it's good this time